app/assets/javascripts/discourse/tests/acceptance/share-topic-test.js

```diff
@@ -59,14 +59,19 @@ acceptance("Share and Invite modal", function (needs) {
 
   test("Post date link", async function (assert) {
     await visit("/t/short-topic-with-two-posts/54077");
-    await click("#post_2 .post-info.post-date");
     assert.ok(
       query("#post_2 .post-info.post-date a").href.endsWith(
         "/t/short-topic-with-two-posts/54077/2?u=eviltrout"
       )
     );
 
+    await click("#post_2 a.post-date");
     assert.ok(exists(".share-topic-modal"), "it shows the share modal");
+    assert.deepEqual(
```

Comment:
```suggestion
    assert.equal(
```
(since it's just a string)

Reply:
Ah, yes, thanks!